ezgui/src/runner.rs

```diff
@@ -39,7 +39,7 @@ pub(crate) struct State<G: GUI> {
 
 impl<G: GUI> State<G> {
     // The bool indicates if the input was actually used.
-    fn event(&mut self, mut ev: Event, prerender: &Prerender) -> (EventLoopMode, bool) {
+    fn event(&mut self, mut ev: Event, prerender: &Prerender) -> (EventCtx, bool) {
```

Comment:
@dabreegster  
I'm getting the following rustc output.
```
   Compiling ezgui v0.1.0 (/Users/javednissar/Documents/Development/abstreet/ezgui)
error[E0623]: lifetime mismatch
   --> ezgui/src/runner.rs:125:25
    |
42  |     fn event(&mut self, mut ev: Event, prerender: &Prerender) -> (EventCtx, bool) {
    |                                                   ----------     ----------------
    |                                                   |
    |                                                   this parameter and the return type are declared with different lifetimes...
...
125 |             Ok(pair) => pair,
    |                         ^^^^ ...but data from `prerender` is returned here

error: aborting due to previous error

For more information about this error, try `rustc --explain E0623`.
error: could not compile `ezgui`.

To learn more, run the command again with --verbose.
```
I'm a bit unsure with regard to how I should manage the lifetime mismatch. Any suggestions?

Reply:
Can you return `ctx.updates_requested` here and just let the `EventCtx` get destroyed?